arghadipmanna101/Flipkart_Clone
# I did changes in sell-online

Related Issue
[#761 ]

Description

This PR enhances the "Sell Online" feature by updating the Seller Success Stories section on the Flipkart Seller Hub page. The following changes have been made:

 Type of PR

- [✔️ ] Feature enhancement
- [ ] Other (specify): _______________

 **Screenshots / videos (if applicable)**

Before:
![SellOnline_Before](https://github.com/arghadipmanna101/Flipkart_Clone/assets/97039791/b20827f7-b2af-4657-8dd7-d3b8ca15329f)


After:
![SellOnline_after(1)](https://github.com/arghadipmanna101/Flipkart_Clone/assets/97039791/e008e84a-54b6-4f61-ae4a-694ed9b94914)

![SellOnline_after(2)](https://github.com/arghadipmanna101/Flipkart_Clone/assets/97039791/76fa5916-a75c-4b37-ab91-5c5564eef944)

Video:

https://github.com/arghadipmanna101/Flipkart_Clone/assets/97039791/321908a4-f7df-4b32-9604-660856e058e7





 Checklist:
- [✔] I have performed a self-review of my code
- [✔] I have read and followed the Contribution Guidelines.
- [✔] I have tested the changes thoroughly before submitting this pull request.
- [✔] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [✔] I have commented my code, particularly in hard-to-understand areas.


How to Test:

- Navigate to the Flipkart Seller Hub page.
- Scroll to the Seller Success Stories section.
- Verify that the carousel displays correctly with the seller images and success stories.
- Check the responsiveness and styling of the carousel across different devices.


Additional context:

1. Ensure that the images (John_doe.jpg and Jane_smith.png) are correctly placed in the img folder.
2. Verify that the carousel navigation works smoothly.
